.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "mocha": true,
+        "jasmine": true
+    },
+    "plugins": [
+        "html"
```

Comment:
I guess this is what that plugin in `package.json` is for.  Is the `html` misleading since we only have console output?